app/controllers/idv/review_controller.rb

```diff
@@ -66,6 +66,7 @@ def create
         success: true,
         deactivation_reason: idv_session.profile.deactivation_reason,
       )
+      log_irs_tmx_fraud_check_event
```

Comment:
I think if we can move this to the verify step, or whatever handles the TMX response directly, that might be better? then we don't have to dig in to the proofing components, we can check the TMX result directly?

Reply:
that is a good point, what about instead the VerifyInfoController that reads the data? https://github.com/18F/identity-idp/blob/main/app/controllers/idv/verify_info_controller.rb#L251 (and also the VerifyStep, until that is removed too)

Reply:
def log_irs_tmx_fraud_check_event
      return unless FeatureManagement.proofing_device_profiling_collecting_enabled?
      threatmetrix_review_pending = !idv_session.profile.active && idv_session.profile.threatmetrix_review_pending?
      failure_reason = { deactivation_reason: ['threatmetrix_review_pending'] } if threatmetrix_review_pending
      irs_attempts_api_tracker.idv_tmx_fraud_check(
        success: !threatmetrix_review_pending,
        failure_reason: failure_reason,
      )
end

Reply:
`def log_irs_tmx_fraud_check_event
      return unless FeatureManagement.proofing_device_profiling_collecting_enabled?
      threatmetrix_review_pending = !idv_session.profile.active && idv_session.profile.threatmetrix_review_pending?
      failure_reason = { deactivation_reason: ['threatmetrix_review_pending'] } if threatmetrix_review_pending
      irs_attempts_api_tracker.idv_tmx_fraud_check(
        success: !threatmetrix_review_pending,
        failure_reason: failure_reason,
      )
    end`

Reply:
```suggestion
     def log_irs_tmx_fraud_check_event
      return unless FeatureManagement.proofing_device_profiling_collecting_enabled?
      threatmetrix_review_pending = !idv_session.profile.active && idv_session.profile.threatmetrix_review_pending?
      failure_reason = { deactivation_reason: ['threatmetrix_review_pending'] } if threatmetrix_review_pending
      irs_attempts_api_tracker.idv_tmx_fraud_check(
        success: !threatmetrix_review_pending,
        failure_reason: failure_reason,
      )
    end
```

Reply:
`
def log_irs_tmx_fraud_check_event
      return unless FeatureManagement.proofing_device_profiling_collecting_enabled?
      threatmetrix_review_pending = !idv_session.profile.active && idv_session.profile.threatmetrix_review_pending?
      failure_reason = { deactivation_reason: ['threatmetrix_review_pending'] } if threatmetrix_review_pending
      irs_attempts_api_tracker.idv_tmx_fraud_check(
        success: !threatmetrix_review_pending,
        failure_reason: failure_reason,
      )
    end
`

Reply:
Thanx https://github.com/18F/identity-idp/blob/main/app/controllers/idv/verify_info_controller.rb#L394 maybe i can add a logging event in this line, here I can get TMX review_status/result directly